etc/config-template.toml

```diff
@@ -418,6 +418,10 @@
 # Pick target size of each level dynamically.
 # dynamic-level-bytes = true
 
+# Optimizes bloom filters. If true, RocksDB won't create bloom filters for the max level of
+# the LSM to reduce metadata that should fit in RAM.
+# optimize-filters-for-hits = true
```

Comment:
I suggest that you should add a comment to enable this for Raw mode.

Reply:
Done.